api/tests/integration/migrations/utils.js

```diff
@@ -166,6 +168,9 @@ const switchToTestDbs = () => {
   db.sentinel = new PouchDB(
     realSentinelDb.name.replace(/medic-sentinel$/, 'medic-sentinel-test')
   );
+  db.users = new PouchDB(
+    realUsersDb.name.replace(/_users$/, 'users-test')
```

Comment:
@dianabarsan I spent a couple hours digging on this today and just want to be sure I am on the right path before investing more time into this.  I agree that it is very not ideal to use the local `COUCH_URL` instance and it would be better to start one in Docker.  After poking at this for awhile, it seems like there are two "viable" approaches here:

1. Move the tests down to `tests/integration/api` and leverage all of the e2e tooling for spooling docker containers.  This seems simple at first, but it is going to be non-trivial to just spin up the Couch containers (and not the rest of the services).  Even if we decide we don't care and will just spin up everything, it still is kinda weird since the tests in  `tests/integration/api`  will be directly referencing the migration script files in `api/src/migrations` to execute them.
2. Keep the tests in `api/tests/integration`, but roll our own custom docker tooling for spinning up Couch that is bespoke to those tests.  This would still need to (I guess?) account for using the freshly generated Couch images, but could maybe avoid docker-compose and clustered couch nodes...

Is either of these close to what you were thinking?

Reply:
I think we should run these integration tests over the correct stack. 
I've had this issue opened two years ago :))  https://github.com/medic/cht-core/issues/7499
I _don't_ think it should be part of this PR. So we could leave it as-is and address it when we move the tests.